Artesian/Artesian.SDK/Dto/DeleteCurveData/DeleteCurveData.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Ark.Tools.Nodatime;
+
+using MessagePack;
+using NodaTime;
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Artesian.SDK.Dto
+{
+    /// <summary>
+    /// The curve data for a delete command.
+    /// </summary>
+    [MessagePackObject]
+    public class DeleteCurveData
+    {
+        /// <summary>
+        /// The default constructor
+        /// </summary>
+        public DeleteCurveData()
+        {
+        }
+
+        /// <summary>
+        /// The constructor with id
+        /// </summary>
+        public DeleteCurveData(MarketDataIdentifier id)
+        {
+            ID = id;
+        }
+
+        /// <summary>
+        /// The constructor with id and version
+        /// </summary>
+        public DeleteCurveData(MarketDataIdentifier id, LocalDateTime version)
+        {
+            ID = id;
+            Version = version;
+        }
+
+        /// <summary>
+        /// The Market Data Identifier
+        /// </summary>
+        [Required]
+        [MessagePack.Key(0)]
+        public MarketDataIdentifier ID { get; set; }
+
+        /// <summary>
+        /// The Version to operate on
+        /// </summary>
+        [MessagePack.Key(1)]
+        public LocalDateTime? Version { get; set; }
+
+        /// <summary>
+        /// The timezone. Must be the OriginalTimezone or, when Hourly, must be "UTC".
+        /// </summary>
+        [Required]
+        [MessagePack.Key(2)]
+        public string Timezone { get; set; }
+
+        /// <summary>
+        /// The Range start and end to be deleted.
+        /// </summary>
+        [Required]
+        [MessagePack.Key(3)]
+        public LocalDateTimeRange Range { get; set; }
```

Comment:
I've missed this on server side, but LocalDateTimeRange is not MsgPack decorated. I'm not even sure it works.

Please use Start/End approach as in all other cases (e.g. CurveRange) for APIs